CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/gui/VisualGraphTopComponent.java

```diff
@@ -549,6 +551,11 @@ public void componentClosed() {
             System.gc();
         }
 
+
+        if (GraphManager.getDefault().getAllGraphs().isEmpty()) {
```

Comment:
Happy to approve as is but curios to know if this would be better placed before we call the `System.gc()` ?

Reply:
Ive made this change, logically I think it makes sense. But I didn't see any difference in behaviour when I tested this in the profiler.

MY guess is that because GC is being called from within this class the stuff that it later garbage collects if you click the button isn't yet available for GC because this class refers to it.